### PR TITLE
NGmerge 0.2 (new formula)

### PR DESCRIPTION
**Pull request by gaberoo**
- [x] Have you followed the [guidelines for contributing](https://github.com/brewsci/homebrew-bio/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/brewsci/homebrew-bio/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source FORMULA`, where `FORMULA` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict FORMULA` (after doing `brew install FORMULA`)?

-----

### CODE REVIEW

File: Formula/ngmerge.rb [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  desc "Merging paired-end reads and removing adapters"
+  homepage "https://github.com/jsh58/NGmerge"
+  url "https://github.com/jsh58/NGmerge/archive/v0.2.tar.gz"
+  sha256 "9d0410ba48f50209e7e652419b3e5d485f515b71ef535f96151cce2dbfdea0de"
+  head "https://github.com/jsh58/NGmerge.git"
+
+  fails_with :clang # needs OpenMP
+
+  depends_on "gcc" if OS.mac?
+  depends_on "zlib" unless OS.mac?
```
Comment (sjackman) [inline Formula/ngmerge.rb]:
```suggestion
  if OS.mac?
    depends_on "gcc"
  else
    depends_on "zlib"
  end
```

File: Formula/ngmerge.rb [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  # cite Gaspar_2018: "https://doi.org/10.1186/s12859-018-2579-2"
+  desc "Merging paired-end reads and removing adapters"
+  homepage "https://github.com/jsh58/NGmerge"
+  url "https://github.com/jsh58/NGmerge/archive/v0.2.tar.gz"
+  sha256 "9d0410ba48f50209e7e652419b3e5d485f515b71ef535f96151cce2dbfdea0de"
+  head "https://github.com/jsh58/NGmerge.git"
+
+  fails_with :clang # needs OpenMP
+
+  depends_on "gcc" if OS.mac?
```
Comment (sjackman) [inline Formula/ngmerge.rb]:
```suggestion
```

**Comment by sjackman** [comment]
Have you seen `fastp`? I'm fine with having this one as well if you'd like both.

**Comment by gaberoo** [comment]
I've never used `fastp`, looks good too though.

**Pull request closed without merging**